catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -60,7 +62,19 @@
  */
 public class GeoJsonInputTransformer implements InputTransformer {
 
-  static final String ISO_8601_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  static final DateTimeFormatter ISO_8601_DATE_FORMAT =
+      new DateTimeFormatterBuilder()
+          .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+          .optionalStart()
+          .appendOffset("+HH:MM", "+00:00")
+          .optionalEnd()
+          .optionalStart()
+          .appendOffset("+HHMM", "+0000")
+          .optionalEnd()
+          .optionalStart()
+          .appendOffset("+HH", "Z")
+          .optionalEnd()
+          .toFormatter();
```

Comment:
❓ Is this not simply covered by `DateTimeFormatter.ISO_DATE_TIME`?

Reply:
That is what I tried first. `ISO_DATE_TIME` will match `+01:00` but will not match what we accepted before which was `+0100` and it also does not support `Z`. This is the best way I found to support all 3 without adding an external dependency.

Reply:
Not sure about the `+0100` part but I have definitely used it with `Z` before and can confirm that it works.  However, if there are other formats that are not covered by `DateTimeFormatter.ISO_DATE_TIME` and are required to be supported, perhaps the tests should be updated to include those?

Reply:
Looks like `+0100` is a non-standard form which is why the Java formatter does not support it. Sadly it is the format we have supported since the creation of this transformer so we will probably have to continue to support it for backwards compatibility.